TSPL.docc/LanguageGuide/OpaqueTypes.md

```diff
@@ -1,20 +1,28 @@
-# Opaque Types
+# Opaque and Boxed Types
 
 Hide implementation details about a value's type.
 
-A function or method with an opaque return type
-hides its return value's type information.
-Instead of providing a concrete type as the function's return type,
-the return value is described in terms of the protocols it supports.
+Swift provides two ways to hide details about a value's type:
+opaque types and boxed protocol types.
 Hiding type information
 is useful at boundaries between
 a module and code that calls into the module,
 because the underlying type of the return value can remain private.
-Unlike returning a value whose type is a protocol type,
-opaque types preserve type identity ---
+
+A function or method that returns an opaque type
```

Comment:
```suggestion
A function or method that returns an *opaque type*
```

Reply:
The lack of new-term italics here was intentional — this is the first mention of the term, but it isn't actually defined here. Did you still want me to add it?

Reply:
Please move the new term italics to the first mention.